src/mixins/observable.mixin.ts

```diff
@@ -142,4 +144,10 @@
     once: once,
     off: off,
   };
+  fabricHeader.Observable = {
+    fire: fire,
+    on: on,
+    once: once,
+    off: off,
+  }
```

Comment:
this is some bullshit i did when i was half asleep.
I didn't even remember doing it.
Going to delete

Reply:
I saw that in #8103 and removed that there as well

Reply:
seems like i added it here?

Reply:
The diff is too harsh.